config_example.sh

```diff
@@ -593,7 +593,13 @@ set -x
 # Note - MIRROR_OLM is currently not supported when MIRROR_COMMAND=oc-mirror, it
```

Comment:
Should we remove/change this note?

Reply:
By the way I tested and `MIRROR_COMMAND=oc-mirror` works **only** for deployments via Agent Installer. It's another story that at some point we need to move everything to use `oc-mirror`, but now even if we wanted, we can't run our flow with it

Reply:
Ah, fair enough. I had it in my head that we needed to change commands too, but I guess that is not the case.

Reply:
The default for `MIRROR_COMMAND` is `oc-adm` so as long as this doesn't change, we are safe